--- config/Makefile.am.org	Sat Apr  1 19:13:45 2000
+++ config/Makefile.am	Sat Apr  1 19:13:56 2000
@@ -4,5 +4,5 @@
 EXTRA_DIST= xroadmaker.conf 
 
 install-data-am : 
-	install -d -m 755 $(prefix)/share/$(PACKAGE)/config
-	install -m 644 xroadmaker.conf $(prefix)/share/$(PACKAGE)/config
+	install -d -m 755 $(DESTDIR)$(prefix)/share/$(PACKAGE)/config
+	install -m 644 xroadmaker.conf $(DESTDIR)$(prefix)/share/$(PACKAGE)/config
--- libs/Makefile.am.org	Sun Mar 19 18:01:37 2000
+++ libs/Makefile.am	Sat Apr  1 19:14:35 2000
@@ -4,6 +4,6 @@
 EXTRA_DIST= standard.compdesclib  dil.layoutlib  pin.spotlib  stdcomponent.set 
 
 install-data-am : 
-	install -m 755 -d  $(prefix)/share/$(PACKAGE)/libs
-	install -m 644 $(EXTRA_DIST) $(prefix)/share/$(PACKAGE)/libs
+	install -m 755 -d  $(DESTDIR)$(prefix)/share/$(PACKAGE)/libs
+	install -m 644 $(EXTRA_DIST) $(DESTDIR)$(prefix)/share/$(PACKAGE)/libs
 
--- pixmaps/Makefile.am.org	Sun Mar 12 12:04:10 2000
+++ pixmaps/Makefile.am	Sat Apr  1 19:15:03 2000
@@ -3,5 +3,5 @@
 EXTRA_DIST=  about.xpm endok.xpm road.xpm track.xpm all.xpm full.xpm roadselect.xpm trackselect.xpm attachpoint.xpm invert.xpm route.xpm undraw.xpm bycolor.xpm mainwindow.xpm  splashscreen.xpm zoomin.xpm circuit.xpm move.xpm startok.xpm zoomout.xpm component.xpm none.xpm subfunction.xpm componenttool.xpm  notfull.xpm subfunctiontool.xpm draw.xpm paint.xpm trace.xpm 
 
 install-data-am : 
-	install -d -m 755 $(prefix)/share/pixmaps/$(PACKAGE)
-	install -m 644 $(EXTRA_DIST) $(prefix)/share/pixmaps/$(PACKAGE)
+	install -d -m 755 $(DESTDIR)$(prefix)/share/pixmaps/$(PACKAGE)
+	install -m 644 $(EXTRA_DIST) $(DESTDIR)$(prefix)/share/pixmaps/$(PACKAGE)
--- Makefile.am.org	Sun Mar 12 12:26:21 2000
+++ Makefile.am	Sat Apr  1 19:15:31 2000
@@ -4,7 +4,7 @@
 EXTRA_DIST = README INSTALL ChangeLog USAGE COPYING TODO
 
 install-data-am : 
-	install -m 755 -d  $(prefix)/share/$(PACKAGE)
+	install -m 755 -d  $(DESTDIR)$(prefix)/share/$(PACKAGE)
 
 rpm : $(PACKAGE)-$(VERSION).tar.gz
 	cp $(PACKAGE)-$(VERSION).tar.gz /usr/src/RPM/SOURCES
